.vib/wordpress/cypress/cypress/fixtures/admins.json

```
@@ -0,0 +1,4 @@
+{
+  "wordpressFirstName": "TestName",
+  "wordpressLastName": "TestLastName"
```

`admins` should define an array of admin users, so that if the tests are ever expanded, one can add another entry:

```suggestion
  "defaultAdmin": {
    "firstName": "TestName",
    "lastName": "TestLastName"
  }
```

---

True. I applied this suggestion, and I also simplified the test a bit since I noticed WordPress had another release with different behaviour from the SMTP product.
Since I saw this happens often, I made this test even more minimalistic.